code/game/machinery/computer/prisoner.dm

```diff
@@ -14,89 +14,83 @@
 	var/stop = 0.0
 	var/screen = 0 // 0 - No Access Denied, 1 - Access allowed
 
+/obj/machinery/computer/prisoner/attack_ai(var/mob/user)
+	return src.attack_hand(user)
```

Comment:
This is generally handled at `/machinery` level in a sufficient manner, and overriding to attack_hand won't do what you'd expect. This is likely one of the few `attack_ai` overrides I didn't kill due to the relative pathing foiling my searches.